packages/components/psammead-navigation/src/index.test.jsx

```diff
@@ -21,6 +21,9 @@ const navigationUlComponent = (
           currentPageText="Current page"
           service="news"
           data-navigation="test_navigation"
+          brandForegroundColour="#FDFDFD"
```

Comment:
Throughout these tests I'd imagine the colours should refer to constants rather than being hardcoded?

Reply:
What benefit would that give?